azteco.md

```diff
@@ -0,0 +1,196 @@
+
+If you need a Bitcoin wallet, consider the following options (Avoid Azteco's recommendations):
+
+**On-chain wallets:**
+- [Envoy](https://foundationdevices.com/envoy/) – Android/iOS
+- [Samourai Wallet](https://samouraiwallet.com/) – Android Only
+- [Sparrow Wallet](https://sparrowwallet.com/) – Desktop
+
+**Lightning wallets:**
+- [Zeus](https://zeusln.app/) – Android/iOS
+- [Phoenix](https://phoenix.acinq.co/) – Android/iOS
+- [Blixt](https://blixtwallet.github.io/) – Android/iOS
+
+## Buy Vouchers
+
+*Note that Azteco offers lightning and on-chain vouchers. In this guide, we will buy an on-chain voucher.*
+
+### Physical Vendors
+The **best** way to acquire your Azteco voucher is through its network of vendors. Check Azteco [official resellers' map](https://azte.co/find_a_vendor) and see if you can trade cash for sats around you. This is the most private way to acquire vouchers as you can pay with cash and bypass request for identification.
+
+### Order online from Azteco
+Azteco offers a hybrid online and in-person service, available in a variety of countries, that allows you to purchase Bitcoin vouchers. The service provides numerous payment methods, including cash, bank transfer, and credit card. Azteco has [cash-in locations](https://azte.co/pay-in-person) across several countries including Brazil, Colombia, Mexico, Peru, Chile, Costa Rica, South Africa, Ghana, Uganda, Zambia, and Ireland.
+
+Start by navigating to the [Pay-in-person](https://azte.co/pay-in-person) page. This page will provide you with all the necessary instructions to order your voucher online. You need to fill-up a form in order to get a bar code. Once you have this bar code you can go to any of the Azteco-affiliated physical locations. This physical vendor will scan your barcode and collect your payment. The vendor will communicate this transaction to Azteco. Once payment is confirmed, Azteco emails you the voucher.
+
+### Online resellers
+If there aren't any Azteco vendors in your area, you can purchase a voucher online from [approved retailers](https://help.azte.co/article/8-buy-a-voucher-approved-retailers) like G2A or Kinguin. However, it's important to remember that online purchases will require an email address and potentially less private payment methods compared to cash transactions.
+
+If you're keen on maintaining your privacy while shopping online, here are some strategies to consider:
+
+- Consider using an [email alias](https://simplelogin.io/). This tool allows you to disguise your actual email address, adding an extra layer of privacy.
+- Purchase a [Paysafecard](www.paysafecard.com) at a local shop and use it for your online transactions. This way, you won't need to use your credit card information online.
+- If possible, ask a friend or family member to purchase the voucher on your behalf, and reimburse them in cash. This could help to preserve your anonymity.
```

Comment:
*Just make sure they are aware of the trade-offs in doing so

Reply:
Good point. Also, since you are much better at this than me, do you have any other recommendations or tips for this part?

Reply:
Honestly I think you've covered the basics well. Buying one of these online is going to be the most difficult part to do privately, and the suggestion you've input sounds like it could be a good alternative. Though I've not tried it myself.

Only thing to add here is the caveat around asking friends and family to purchase on your behalf. Just mention that you should ensure they understand the trade-offs as the bitcoin purchase could be tied to them.

Reply:
Noted, I will add this on the trade-offs. Brilliant!